MPF.Core/InfoTool.cs

```diff
@@ -425,54 +485,26 @@ internal static bool GetPlayStationExecutableInfo(string? drivePath, out string?
             if (!Directory.Exists(drivePath))
                 return false;
 
-            // Get the two paths that we will need to check
-            string psxExePath = Path.Combine(drivePath, "PSX.EXE");
-            string systemCnfPath = Path.Combine(drivePath, "SYSTEM.CNF");
-
-            // Try both of the common paths that contain information
-            string? exeName = null;
-
-            // Read the CNF file as an INI file
-            var systemCnf = new IniFile(systemCnfPath);
-            string bootValue = string.Empty;
+            // Get the executable name
+            string? exeName = GetPlayStationExecutableName(drivePath);
 
-            // PlayStation uses "BOOT" as the key
-            if (systemCnf.ContainsKey("BOOT"))
-                bootValue = systemCnf["BOOT"];
-
-            // PlayStation 2 uses "BOOT2" as the key
-            if (systemCnf.ContainsKey("BOOT2"))
-                bootValue = systemCnf["BOOT2"];
-
-            // If we had any boot value, parse it and get the executable name
-            if (!string.IsNullOrEmpty(bootValue))
-            {
-                var match = Regex.Match(bootValue, @"cdrom.?:\\?(.*)", RegexOptions.Compiled);
-                if (match.Groups.Count > 1)
-                {
-                    // EXE name may have a trailing `;` after
-                    // EXE name should always be in all caps
-                    exeName = match.Groups[1].Value
-                        .Split(';')[0]
-                        .ToUpperInvariant();
+            // If no executable found, we can't do this part
+            if (exeName == null)
+                return false;
 
-                    // Serial is most of the EXE name normalized
-                    serial = exeName
-                        .Replace('_', '-')
-                        .Replace(".", string.Empty);
+            // EXE name may have a trailing `;` after
+            // EXE name should always be in all caps
+            exeName = exeName
+                .Split(';')[0]
+                .ToUpperInvariant();
 
-                    // Some games may have the EXE in a subfolder
-                    serial = Path.GetFileName(serial);
-                }
-            }
-
-            // If the SYSTEM.CNF value can't be found, try PSX.EXE
-            if (string.IsNullOrEmpty(exeName) && File.Exists(psxExePath))
-                exeName = "PSX.EXE";
+            // Serial is most of the EXE name normalized
+            serial = exeName
+                .Replace('_', '-')
+                .Replace(".", string.Empty);
 
-            // If neither can be found, we return false
-            if (string.IsNullOrEmpty(exeName))
-                return false;
+            // Some games may have the EXE in a subfolder
+            serial = Path.GetFileName(serial);
```

Comment:
I think this is handled in `GetPlayStationExecutableName` now, at least from how I'm reading this.

Reply:
I didn't want to think about all possible cases so left it just-in-case, e.g. splitting at `;` created a new file path somehow. If you're happy with it being removed, I can remove it.

Reply:
Splitting shouldn't cause an issue with the path. The `;` isn't a known directory separator or anything and if it's reassembled the same way, you're golden.